kelvin-calculator/src/components/Calculator.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component, useEffect, useState } from 'react'
+import {evaluate} from "mathjs";
+import NumberFormat from 'react-number-format'
+import '../components/Calculator.css'
+
+export default class Calculator extends Component {
+
+
+    
+        state = {
```

Comment:
Having two states to hold displayValue and prevValue would have prevented your display from acting the way it did